pkgs/development/libraries/openmpi/default.nix

```diff
@@ -29,6 +30,12 @@ in stdenv.mkDerivation rec {
     sha256 = "0ms0zvyxyy3pnx9qwib6zaljyp2b3ixny64xvq3czv3jpr8zf2wh";
   };
 
+  # TODO: Remove these, when a new release gets out
+  patches = [
+    (fetchpatch { name = "ucx-1.8-compat-1"; url = "https://github.com/open-mpi/ompi/commit/526775dfd7ad75c308532784de4fb3ffed25458f.patch"; sha256 = "1zfr7ym4303dsnnjqfc1qbnfs0izd8bazxfl57p3dhqsiznd734r"; })
```

Comment:
I am still confused why there are patches for UCX 1.8 when are building against UCX 1.7.

Reply:
Take a look at the two commit messages:

https://github.com/open-mpi/ompi/commit/526775dfd7ad75c308532784de4fb3ffed25458f
https://github.com/open-mpi/ompi/commit/a3026c016a6a8be379f62585b6ddc070175c8106

The first one adds support and the second replaces the version bound `version > 1.7` with `version >= 1.7`. They are named that way, to make it clear, that they can be removed together.

Reply:
The way I understand the patches the first one fixes a problem with ucx/master (correct me if I am wrong). Since we are not building against ucx 1.8 (or current master) that patch is not needed right now. We should remove it for now.